apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -212,7 +212,7 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
     }
 
     return Promise.resolve()
-        .then(() => buzzer.play(SONG_CHARGE, 104))
+        .then(() => buzzer.playSong(SONG_CHARGE, 104))
```

Comment:
Since the `play` method isn't going away, is there an advantage to changing this call to use the alias?